utils/getInvoices/tests/getInvoices.electricity.test.ts

```diff
@@ -6,143 +6,551 @@ import { ServiceType } from '@utils/constants'
+
+      expect(invoices).not.toContainEqual({
+        type: ServiceType.Electricity,
+      })
+    })
+
+    it('should NOT load when company = { ...: NaN }', () => {
+      const company: Partial<IRealestate> = {
+        discount: NaN,
+      }
+
+      const invoices = getInvoices({
+        company,
+      })
+
+      expect(invoices).not.toContainEqual({
+        type: ServiceType.Electricity,
+      })
+    })
+
+    it('should NOT load when company = { ...: null }', () => {
+      const company: Partial<IRealestate> = {
+        discount: null,
+      }
+
+      const invoices = getInvoices({
+        company,
+      })
+
+      expect(invoices).not.toContainEqual({
+        type: ServiceType.Electricity,
+      })
+    })
+
+    it('should NOT load when company = { ...: undefined }', () => {
+      const company: Partial<IRealestate> = {
+        discount: undefined,
+      }
+
+      const invoices = getInvoices({
+        company,
+      })
+
+      expect(invoices).not.toContainEqual({
+        type: ServiceType.Electricity,
+      })
     })
   })
 
-  it('should NOT load Electricity from service without electricity', () => {
-    const company: Partial<IRealestate> = {}
-    const service: Partial<IService> = {
-      electricityPrice: null,
-    }
-    const payment: Partial<IPayment> = {}
-    const prevPayment: Partial<IPayment> = {}
+  describe('props: { payment }', () => {
+    it('should NOT load when payment = null', () => {
+      const payment: Partial<IPayment> = null
+
+      const invoices = getInvoices({
+        payment,
+      })
 
-    const invoices = getInvoices({
-      company,
-      service,
-      payment,
-      prevPayment,
+      expect(invoices).not.toContainEqual({
+        type: ServiceType.Electricity,
+      })
     })
+    it('should NOT load when payment = { invoice: null }', () => {
+      const payment: Partial<IPayment> = {
+        invoice: null,
+      }
 
-    expect(invoices).not.toContainEqual(
-      expect.objectContaining({ type: ServiceType.Electricity })
-    )
+      const invoices = getInvoices({
+        payment,
+      })
+
+      expect(invoices).not.toContainEqual({
+        type: ServiceType.Electricity,
+      })
+    })
+    it('should NOT load when payment = { invoice: [] }', () => {
+      const payment: Partial<IPayment> = {
+        invoice: [],
+      }
+
+      const invoices = getInvoices({
+        payment,
+      })
+
+      expect(invoices).not.toContainEqual({
+        type: ServiceType.Electricity,
+      })
+    })
+    it('should NOT load when payment = { invoice: [Electricity] }', () => {
+      const payment: Partial<IPayment> = {
+        invoice: [
+          {
+            type: ServiceType.Electricity,
+            price: 10,
+            sum: 100,
+          },
+        ],
+      }
+
+      const invoices = getInvoices({
+        payment,
+      })
+
+      expect(invoices).not.toContainEqual({
+        type: ServiceType.Cleaning,
+      })
+    })
```

Comment:
This one shoul load Electricity with 0 amount's.

And your is broken because of `type: ServiceType.Cleaning,`

Reply:
Also, sorry for my mistake in example tests, because this test case is succesed even if it's shouldn't.

Replace
```
expect(invoices).not.toContainEqual({
  type: ServiceType.Type,
})
```
with

```
expect(invoices).not.toContainEqual(
  expect.objectContaining({ type: ServiceType.Type})
)
```

Do this to all completed tests, please. water, water-part, cleaning, electricity. DM me if some test cases would broke